config.rb

```diff
@@ -0,0 +1,73 @@
+###
+# Page options, layouts, aliases and proxies
+###
+
+# Per-page layout changes:
+#
+# With no layout
+# page "/path/to/file.html", :layout => false
+#
+# With alternative layout
+# page "/path/to/file.html", :layout => :otherlayout
+#
+# A path which all have the same layout
+# with_layout :admin do
+#   page "/admin/*"
+# end
+
+# Proxy pages (https://middlemanapp.com/advanced/dynamic_pages/)
+# proxy "/this-page-has-no-template.html", "/template-file.html", :locals => {
+#  :which_fake_page => "Rendering a fake page with a local variable" }
+
+###
+# Helpers
+###
+
+# Automatic image dimensions on image_tag helper
+# activate :automatic_image_sizes
+
+# Reload the browser automatically whenever files change
+# configure :development do
+#   activate :livereload
+# end
+
+# Methods defined in the helpers block are available in templates
+# helpers do
+#   def some_helper
+#     "Helping"
+#   end
+# end
+
+set :source, "contrib"
+
+set :css_dir, 'assets/stylesheets'
+
+set :js_dir, 'assets/javascripts'
+
+set :images_dir, 'assets/images'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.